includes/class-apple-news.php

```diff
@@ -544,19 +544,19 @@ public function create_default_theme() {
 	 * otherwise the same as the existing value in the database will cause a failure
 	 * during post save.
 	 *
-	 * @see \update_metadata
-	 *
-	 * @param null|bool $check      Whether to allow updating metadata for the given type.
-	 * @param int       $object_id  Object ID.
-	 * @param string    $meta_key   Meta key.
+	 * @param bool|null $check Whether to allow updating metadata for the given type.
+	 * @param int       $object_id Object ID.
+	 * @param string    $meta_key Meta key.
 	 * @param mixed     $meta_value Meta value. Must be serializable if non-scalar.
 	 * @param mixed     $prev_value Optional. If specified, only update existing.
-	 * @return null|bool True if the conditions are ripe for the fix, otherwise the existing value of $check.
+	 *
+	 * @return null|bool True, if the conditions are ripe for the fix, otherwise the existing value of $check.
+	 * @see \update_metadata
 	 */
-	public function filter_update_post_metadata( $check, $object_id, $meta_key, $meta_value, $prev_value ) {
+	public function filter_update_post_metadata( ?bool $check, int $object_id, string $meta_key, mixed $meta_value, mixed $prev_value ): ?bool {
```

Comment:
I'd expect typecasting those variables a bit no, right?! Since technically, any plugin can hook into `update_post_metadata` and alter the returned type.

Reply:
Good point, I'll remove.